docs/docs/business-logic.mdx

```diff
@@ -78,28 +89,19 @@ slots:
     - type: from_intent
       intent: affirm
       value: true
+      conditions:
+       - active_loop: restaurant_form
+          requested_slot: outdoor_seating
```

Comment:
Nit-picking (could also be my poor eyesight 😂 ): there seems to be an extra whitespace before `requested_slot`, should this be aligned with `active_loop` to make it look pretty? 😄  same comment for L 100.

Reply:
you're totally right will fix =)

Reply:
```suggestion
         requested_slot: outdoor_seating
```